### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by TylerSheffels**
There were small changes I came across doing a fresh dev deploy.

This pr requires adding `host_name` to ES workers in inventories. 
https://github.com/dimagi/commcare-hq/pull/6166

@snopoke @dannyroberts

### CODE REVIEW

File: ansible/roles/ecryptfs/tasks/main.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
   apt: name=ecryptfs-utils state=present
 
 - name: Check for already mounted drive
-  shell: cat /etc/fstab
-  register: fstab_contents
+  shell: cat /etc/mtab
+  register: mtab_contents
```
Comment (dannyroberts) [inline ansible/roles/ecryptfs/tasks/main.yml]: What'd this change? This seemed to be working on india

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 /config/
 .vagrant/
 DimagiKeyStore
+.idea/
```
Comment (benrudolph) [inline .gitignore]: why ignore ideas?

File: ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2 [hunk 3]
```diff
@@ -30,7 +30,7 @@
 # multiple clusters on the same network, make sure you're using unique names.
 #
 # cluster.name: elasticsearch
-cluster.name: {{ elasticsearch_cluster_name }}
+cluster.name: elasticsearch_{{ deploy_env }}
```
Comment (dannyroberts) [inline ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2]: Maybe make this a setting instead. In prod for example i think it's `prodhqes`

File: ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2 [hunk 4]
```diff
@@ -40,10 +40,7 @@ cluster.name: {{ elasticsearch_cluster_name }}
 # from configuring them manually. You can tie this node to a specific name:
 #
 # node.name: "Franz Kafka"
-{% if elasticsearch_node_name %}
-{#- todo: this only makes any sense for single machine setup -#}
-node.name: "{{ elasticsearch_node_name }}"
-{% endif %}
+node.name: "elasticsearch_node_{{ host_name }}"
```
Comment (dannyroberts) [inline ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2]: I think this shouldn't have a prefix, so you can just specify the whole thing as an inventory variable `es_node_name` (on prod it's `hqes0` and `hqes1`)

File: ansible/roles/ecryptfs/tasks/main.yml [hunk 5]
```diff
@@ -39,9 +39,9 @@
...
 - name: Create entry in fstab
   sudo: yes
-  lineinfile: 
+  lineinfile:
     dest: "/etc/fstab"
     regexp: "^/opt/data"
     backup: yes
     line: "{{ mount_configuration.stdout }}"
-  when: "'/opt/data' not in fstab_contents.stdout"
+  when: "'/opt/data' not in mtab_contents.stdout"
```
Comment (benrudolph) [inline ansible/roles/ecryptfs/tasks/main.yml]: Is this needed given the `regexp` clause?

Comment (benrudolph) [inline ansible/roles/ecryptfs/tasks/main.yml]: just a minor thing, realize this isn't code you wrote

**Comment by dannyroberts** [comment]
made a few comments, otherwise looks good!

File: .gitignore (hunk 2 above)
Comment (TylerSheffels) [inline .gitignore]: pycharms leaves workspace information in here

File: ansible/roles/ecryptfs/tasks/main.yml (hunk 1 above)
Comment (TylerSheffels) [inline ansible/roles/ecryptfs/tasks/main.yml]: fstab stores filesystems available to mount, mtab stores the ones actually mounted.  The second time this is run after a restart the line will be in fstab, and skip the mounting process. http://www.linuxquestions.org/questions/linux-newbie-8/difference-between-mtab-and-fstab-924190/

Comment (benrudolph) [inline ansible/roles/ecryptfs/tasks/main.yml]: :+1:

Comment (benrudolph) [inline ansible/roles/ecryptfs/tasks/main.yml]: also, someone should seriously take all the questions on linuxquestions and move the to stackexchange

Comment (dannyroberts) [inline ansible/roles/ecryptfs/tasks/main.yml]: Ok, so you're saying this was always wrong, and it should always have been mtab.

File: ansible/roles/ecryptfs/tasks/main.yml (hunk 5 above)
Comment (TylerSheffels) [inline ansible/roles/ecryptfs/tasks/main.yml]: I think so, the regexp finds the line to replace (if there is a change), but there is no need to replace that line when we are not mounting the drive

**Comment by TylerSheffels** [comment]
@dannyroberts @bennojoy updated

**Comment by dannyroberts** [comment]
fyi just rebased due to https://github.com/dimagi/commcarehq-ansible/pull/71 (but didn't squash any commits :))

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 6]
```diff
@@ -164,7 +164,7 @@ ANALYTICS_IDS = {
 
 AXES_LOCK_OUT_AT_FAILURE = False
 
-LUCENE_ENABLED = True
+LUCENE_ENABLED = False
```
Comment (snopoke) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: @TylerSheffels why did you need to change this? This breaks the search functionality on couchlog.

Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: wow, nice find.

Comment (snopoke) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: @TylerSheffels unless I hear back I'm just going to change this back to True

Comment (TylerSheffels) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: @snopoke Going through this cleanup branch I thought that Lucene was no longer used, and no one else thought it was used either. Sounds like definitely re-enable.

Comment (snopoke) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: 👍

On 4 August 2015 at 15:24, TylerSheffels <notifications@github.com> wrote:

> In ansible/roles/commcarehq/templates/localsettings.py.j2
> <https://github.com/dimagi/commcarehq-ansible/pull/65#discussion_r36186386>
> :
>
> > @@ -164,7 +164,7 @@ ANALYTICS_IDS = {
> >
> >  AXES_LOCK_OUT_AT_FAILURE = False
> >
> > -LUCENE_ENABLED = True
> > +LUCENE_ENABLED = False
>
> @snopoke <https://github.com/snopoke> Going through this cleanup branch I
> thought that Lucene was no longer used, and no one else thought it was used
> either. Sounds like definitely re-enable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcarehq-ansible/pull/65/files#r36186386>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa
